specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json

```diff
@@ -334,6 +334,61 @@
       },
       "type": "object"
     },
+    "ListEdgePoliciesInput": {
```

Comment:
Adding new properties to existing API version is classified as breaking change as per Azure API guideline

Reply:
Such change requires a new api version.

Reply:
@ravbhatnagar This is a new definition, not a new property, we are adding a new action to the entity, no new properties are added.